.travis.yml

```diff
@@ -6,18 +6,21 @@ language: php
 matrix:
     include:
         - php: 5.6
-          env: SCENARIO=php5
+          env: PHPUNIT_VERSION=^5.7
         - php: 7.1
-          env: SCENARIO=php7
+          env: PHPUNIT_VERSION=^6.4
         - php: 7.2
-          env: SCENARIO=php7
+          env: PHPUNIT_VERSION=^6.4
+
+before_install:
+    - if [ "$PHPUNIT_VERSION" != "" ]; then composer require phpunit/phpunit=$PHPUNIT_VERSION; fi
```

Comment:
Instead of using the env variable for phpunit I would use it for the Symfony dependencies. AFAIK the phpunit version is selected by composer directly based on the PHP version used.